installer/create-database.php

```diff
@@ -374,6 +374,7 @@ function populateDatabase($oDB)
             'scale_id' =>  "integer NOT NULL default '0'",
             'same_default' =>  "integer NOT NULL default '0'",
             'relevance' =>  "text",
+            'question_theme_name' => "string(150) NULL",
```

Comment:
Personally i prefer to have $oQuestion->theme_name or $oQuestion->themename since we are in question object.

Reply:
Well. We have many types of themes. This is question theme.

Reply:
As this will be used as foreign key for fetching question_theme_names, maybe it is more explicit (although redundant) to levae it as it is now.

Reply:
But we already are in question … like relevance in question are question relevance …

$oQuestion->themename are foreing key for $theme->name : this is clear enough.

Reply:
But there is no relevance object or table.

Reply:
> $oQuestion->themename are foreing key for $theme->name : this is clear enough.

We are in question, why add question again …

Reply:
because the foreign table is named like that